core/backend/src/IModelHost.ts

```diff
@@ -195,6 +198,9 @@ export interface IModelHostOptions {
    * Will be changed to default to `false` in 5.0.
    */
   allowSharedChannel?: boolean;
+
+  /** @internal */
```

Comment:
Are this and `defaultPullMergeMethod` really supposed to remain `internal`? Currently you provide no way for apps to choose between rebase and merge.

Reply:
Keeping them internal. As @kabentley points out, we can't really allow individual apps to decide. We'll keep the internal APIs temporarily for testing.